common/views/components/MetaUnit/MetaUnit.js

```diff
@@ -1,4 +1,5 @@
 // @flow
+import NextLink from 'next/link';
```

Comment:
I think I've used `<Link>` in the works page – did you chose `NextLink` because we already have the concept of `Link`. Should we update one?